### PR TITLE
Fixes problem for mapping Faoo_fco2_ocn to atm

### PR DESCRIPTION
**Pull request by mvertens**
### Description of changes

The mapping of Faoo_fco2_ocn must have a normalization of 1 and must be mapped with mapconsd NOT mapconsf.

### Specific notes
This was found during the validation of the compset 
`1850_CAM60_CLM50%BGC-CROP_CICE_POP2%ECO_MOSART_SGLC_WW3_BGC%BPRP 
`
at f19_g17 

This also fixes the problem encountered running a B1850 compset with cmeps0.13.10 
Its important to note that current any mapping type for a given source->destination pair can only have one normalization.
The problem was that between ocn->atm the old mapping for Faoo_fco2_ocn was mapconsf and the normalization was 1.
The new error check introduced by @DeniseWorthen found a problem with the So_t between ocn->atm was trying to use a normalization of 'ofrac'.

Contributors other than yourself, if any:

CMEPS Issues Fixed: #204

Are changes expected to change answers?
 - [x] bit for bit
 - [ ] different at roundoff level
 - [ ] more substantial

Any User Interface Changes (namelist or namelist defaults changes)?
 - [ ] Yes
 - [x] No

Testing performed if application target is CESM:(either UFS-S2S or CESM testing is required):
- [ ] (recommended) CIME_DRIVER=nuopc scripts_regression_tests.py
   - machines:
   - details (e.g. failed tests):
- [ ] (recommended) CESM testlist_drv.xml
   - machines and compilers:
   - details (e.g. failed tests):
- [ ] (optional) CESM prealpha test
   - machines and compilers
   - details (e.g. failed tests):
- [x] (other) please described in detail
   - machines and compilers
   - details (e.g. failed tests):
   Verified that the following test ran: SMS_Ld1_Vnuopc.f19_g17.B1850.cheyenne_intel

Testing performed if application target is UFS-coupled:
- [ ] (recommended) UFS-coupled testing
   - description:
   - details (e.g. failed tests):

Testing performed if application target is UFS-HAFS:
- [ ] (recommended) UFS-HAFS testing
   - description:
   - details (e.g. failed tests):

Hashes used for testing:
- [ ] CESM:
  - repository to check out: https://github.com/ESCOMP/CESM.git
  - hash: cesm2_3_beta03  (updated CIME to 71ddf1ced - current master)
- [ ] UFS-coupled, then umbrella repostiory to check out and associated hash:
  - repository to check out:
  - branch:
  - hash:
- [ ] UFS-HAFS, then umbrella repostiory to check out and associated hash:
  - repository to check out:
  - branch:
  - hash:

### CODE REVIEW

File: mediator/med_phases_history_mod.F90 [hunk 1]
```diff
@@ -453,6 +459,10 @@ subroutine med_phases_history_write(gcomp, rc)
           end if
        enddo
 
+       ! Write out areas to history file
+       do n = 1,ncomps
```
Comment (jedwards4b) [inline mediator/med_phases_history_mod.F90:463]: Is something missing here?

Comment (mvertens) [reply mediator/med_phases_history_mod.F90]: Good catch. Forgot to delete that do loop. 
The B1850 test now seems to be running fine.

**Review by DeniseWorthen — APPROVED** [review]
This is fine for ufs-weather, both gnu and intel

**Comment by mvertens** [comment]
@DeniseWorthen - thanks so much for the quick response to this PR. @jedwards4b - if you are okay with this as it is - can you please okay and merge it?

**Review by jedwards4b — APPROVED** [review]
I added the change at med_map_mod.F90 line 820.

**Pull request merged**